RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -66,11 +68,11 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
                                                      const unsigned int mdIndex2,
                                                      const unsigned int mdIndex3,
                                                      const float radius,
-                                                     const float betaIn) {
+                                                     const float betaIn,
+                                                     float (&output)[dnn::t3dnn::kOutputFeatures]) {
```

Comment:
isn't
```suggestion
                                                     float output[dnn::t3dnn::kOutputFeatures]) {
```
equivalent ?

Reply:
I see what you mean, but you do get a compile-time check from the reference form that the array passed to output is exactly kOutputFeatures elements.